service/service.go

```diff
@@ -0,0 +1,102 @@
+package service
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/companieshouse/chs.go/avro"
+	"github.com/companieshouse/chs.go/kafka/consumer/cluster"
+	"github.com/companieshouse/chs.go/log"
+	"github.com/companieshouse/company-search-consumer/upsert"
+)
+
+// Value required to unit test the consumer
+var testHttpResponseValue int
+
+// messageMetadata represents resource change data json unmarshal
+type messageMetadata struct {
+	ResourceKind string `avro:"resource_kind"`
+	ResourceURI  string `avro:"resource_uri"`
+	ContextID    string `avro:"context_id"`
+	ResourceID   string `avro:"resource_id"`
+	Data         string `avro:"data"`
+	Event        Event  `avro:"event"`
+}
+
+// Event - resource changed data event
+type Event struct {
+	FieldsChanged []string `avro:"fields_changed,omitempty"`
+	PublishedAt   string   `avro:"published_at"`
+	Type          string   `avro:"type"`
```

Comment:
These are already defined in https://github.com/companieshouse/chs-go-avro-schemas

Reply:
This has not yet been resolved?